.pylintrc

```diff
@@ -1,10 +1,25 @@
 [MASTER]
 jobs = 0
+extension-pkg-whitelist=numpy,torch
+init-hook="from pylint.config import find_pylintrc; import os, sys; sys.path.append(os.path.dirname(find_pylintrc()))"
 
 [MESSAGES CONTROL]
 disable = missing-class-docstring,
           missing-function-docstring,
-          missing-module-docstring
+          missing-module-docstring,
```

Comment:
Too many disable controls

Reply:
Most disrables have been modified. The rest of the disables takes time to make major changes to the code，The next version iteration will modify it.